WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/CategoryViewHolder.kt

```diff
@@ -1,31 +1,28 @@
 package org.wordpress.android.ui.layoutpicker
 
-import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.modal_layout_picker_category.view.*
-import org.wordpress.android.R
+import org.wordpress.android.databinding.ModalLayoutPickerCategoryBinding
 import org.wordpress.android.login.util.getColorStateListFromAttribute
 import org.wordpress.android.util.getColorFromAttribute
 import org.wordpress.android.util.setVisible
+import org.wordpress.android.util.viewBinding
 
 /**
  * Renders the Layout Category header buttons
  */
-class CategoryViewHolder(internal val parent: ViewGroup) : RecyclerView.ViewHolder(
-        LayoutInflater.from(parent.context).inflate(
-                R.layout.modal_layout_picker_category,
-                parent,
-                false
-        )
-) {
-    val container: View = itemView.category_container
-    val category: TextView = itemView.category
-    val emoji: TextView = itemView.emoji
-    val check: ImageView = itemView.check
+class CategoryViewHolder(
+    internal val parent: ViewGroup,
+    internal val binding: ModalLayoutPickerCategoryBinding =
+            parent.viewBinding(ModalLayoutPickerCategoryBinding::inflate)
+) : RecyclerView.ViewHolder(binding.root) {
+    val container: View = binding.categoryContainer
+    val category: TextView = binding.category
+    val emoji: TextView = binding.emoji
+    val check: ImageView = binding.check
```

Comment:
❓ : I know we initialize the fields at the beginning for caching potentially expensive` View.findViewById(int)` results. Do we need to do this even with the view binding?  Just curious, in case you know 😇.

Reply:
Good question @ashiagr !

I actually have the same feeling and believe that this is not needed. I would avoid using extra fields here and use `binding.xyz` directly when needed, for both `onBind(...)` and `setSelectedStateUI(...)` methods.

PS: Same applies to `LayoutViewHolder`.
PSS: I also have some additional feedback on both those holder classes, which I am going to include in my code review.

Reply:
@ParaskP7 @ashiagr - Just a matter of "how far" do I take the refactoring to meet the original goal of removing synthetic accessors? I can definitely update.

Reply:
Removed in c654b23